### ISSUE TITLE
xpcall arguments

### ISSUE THREAD

**Issue by mrbuds** [Ketho/vscode-wow-api#38]
In 5.1 xpcall has 2 arguments, optionals were added in 5.2
WoW use 5.2 xpcall, i don't know if something can be done about it with vscode-wow-api

**Comment by Ketho** [comment]
I documented it as this but I don't know how to replace the definition from sumneko, will have to ask
```lua
---@param f function
---@param err function
---@vararg any
---@return boolean status
---@return ...
function xpcall(f, err, ...) end
```
![image](https://user-images.githubusercontent.com/1073877/169445534-3a0226c6-3c44-40c4-9df8-29705d53da61.png)

**Comment by Ketho** [comment]
Should be fixed by replacing the LuaLS xpcall in 0.15.0
https://github.com/Ketho/vscode-wow-api/blob/2ae75ed53667b7dae7cf23840b066aca6d505029/Annotations/Lua/basic.lua#L338
